fedimint-server/src/config/api.rs

```diff
@@ -841,6 +905,14 @@ pub fn server_endpoints() -> Vec<ApiEndpoint<ConfigGenApi>> {
                 config.restart_federation_setup().await
             }
         },
+        api_endpoint! {
+            CHECK_BITCOIN_STATUS_ENDPOINT,
+            ApiVersion::new(0, 0),
```

Comment:
Should we bump the minor?

Reply:
Eric said no for authed guardian only apis since the UI has to match the version